Gemfile

```diff
@@ -1,10 +1,12 @@
 source ENV['GEM_SOURCE'] || 'https://artifactory.delivery.puppetlabs.net/artifactory/api/gems/rubygems/'
 
-gem 'beaker', :git => 'https://github.com/puppetlabs/beaker.git', :branch => 'fix/master/BKR-487_preserved_hosts_yml_fix'
```

Comment:
Did the preserved hosts issue for ec2 instances get fixed in core beaker, or will we see this issue again?

Reply:
I thought I remembered this got fix somehow a while back.  I believe it was when we moved to abs.  I should probably test it though.